BACKEND/index.js

```diff
@@ -0,0 +1,132 @@
+const mongoose = require('mongoose');
+let canteenmenu=require("./model.js");
+const bodyParser=require("body-parser")
+const ejs = require("ejs");
+
+
+const express=require('express');
+const app=express();
+const path=require('path');
+const port=80;
+app.set('view engine', 'ejs');
+app.use(express.static("public"));
+
+// import canteenmenu from "./model.js";
+
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
+
+main().catch(err => console.log(err));
+
+async function main() {
+  await mongoose.connect('mongodb://127.0.0.1:27017/canteen');
+    console.log("WE ARE NOW CONNECTED");
+  // use `await mongoose.connect('mongodb://user:password@127.0.0.1:27017/test');` if your database has auth enabled
+}
+// const canteenmenu = new mongoose.Schema({
+//    name: String,
+//     price: Number,
+//     description: String,
+//     id:String
+//   });
+
+  const Menu = mongoose.model('Menu', canteenmenu);
+
+  
+  const menuitems = Menu.find({}).then((err, posts) => {

```

Comment:
Add two more folders routers and controllers, routers folder would have two files user-routes and food-routes, controller would have two files  user-controller and food-controller, Router will handle the routes for the http requests and controller will handle the controller function on each request